include/beman/iterator_interface26/detail/stl_interfaces/config.hpp

```diff
@@ -0,0 +1,49 @@
+// include/beman/optional26/detail/stl_interfaces/config.hpp -*-C++-*-
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// Copyright (C) 2020 T. Zachary Laine
```

Comment:
I suppose this is code from boost.  Just a reminder to check and keep only want we need.

Reply:
This was taken from optional26 so I think it's already been trimmed down.